docs/data-sources/primate-ai.mdx

```diff
@@ -6,24 +6,124 @@ import JSON from './primate-ai-json.md'
 
 ## Overview
 
-Primate AI is a deep residual neural network for classifying the pathogenicity of missense mutations. The method is described in the publication:
+Primate AI is a deep residual neural network for classifying the pathogenicity of missense mutations.
+
+The newer version, PrimateAI-3D, uses a 3D convolutional neural network, to predict protein variant pathogenicity using structural information.
+The model's innovative use of primate sequencing and structural data offers promising insights into variant interpretation and disease gene identification.
+The predictive score range between 0 and 1, with 0 being benign and 1 being most pathogenic.
+
+
+For more details, refer to these publications:
 
 :::info Publication
-Sundaram, L., Gao, H., Padigepati, S.R. et al. Predicting the clinical impact of human mutation with deep neural networks. *Nat Genet* **50**, 1161–1170 (2018). [https://doi.org/10.1038/s41588-018-0167-z](https://doi.org/10.1038/s41588-018-0167-z)
+1. Hong Gao et al. ,The landscape of tolerated genetic variation in humans and primates. *Science* **380**, eabn8153 (2023). [https://doi.org/10.1126/science.abn8197](https://doi.org/10.1126/science.abn8197)
+2. Sundaram, L., Gao, H., Padigepati, S.R. et al. Predicting the clinical impact of human mutation with deep neural networks. *Nat Genet* **50**, 1161–1170 (2018). [https://doi.org/10.1038/s41588-018-0167-z](https://doi.org/10.1038/s41588-018-0167-z)
 :::
 
-## TSV File
+Primate AI is available in two versions based on assembly:
+1. Primate AI 3D: Only available for GRCh38
+1. Primate AI: Only available for GRCh37
 
-### Example
+Both have different file structures, and information. Therefore, they are handled separately:
+
+## Primate AI 3D: GRCh38
+### Parsing
+
+#### CSV File
+
+```scss
+,chr,pos,non_flipped_ref,non_flipped_alt,gene_name,change_position_1based,ref_aa,alt_aa,score_PAI3D,percentile_PAI3D,refseq
+0,chr1,69094,G,A,ENST00000335137.4,2,V,M,0.6169436463713646,0.5200308441794135,NM_001005484.1
+1,chr1,69094,G,C,ENST00000335137.4,2,V,L,0.5557043975591658,0.4271457250214688,NM_001005484.1
+2,chr1,69094,G,T,ENST00000335137.4,2,V,L,0.5557043975591658,0.4271457391722522,NM_001005484.1
+```
+
+From the CSV file, all columns are parsed:
+* `chr`
+* `pos`
+* `ref`
+* `alt`
+* `gene_name`
+* `change_position_1based`
+* `ref_aa`
+* `alt_aa`
+* `score_PAI3D`
+* `percentile_PAI3D`
+* `refseq`
+
+The fields `gene_name` and `refseq` define the Ensembl and RefSeq transcript IDs respectively.
+These transcripts are passed as-is and some of them might be unrecognized/deprecated by RefSeq/Ensembl.
+
+#### Parsing Run
+```text
```

Comment:
Not sure if having the parsing run output is useful, but having the command too would benefit the users. So, let's add the command too.